src/test/java/spoon/support/visitor/java/JavaReflectionTreeBuilderTest.java

```diff
@@ -682,4 +683,14 @@ public boolean matches(CtType element) {
 		assertEquals(true, ctClass.isShadow());
 		assertEquals("foo", ctClass.getMethods().toArray(new CtMethod[0])[0].getSimpleName());
 	}
+
+	@Test
+	public void testExpectedExceptionInInitializerError() {
+		try {
```

Comment:
could you add a comment "contract: ..." in natural language where you describe the intention of the test? Thanks

Reply:
ok